Gruntfile.js

```diff
@@ -197,8 +202,29 @@ module.exports = function(grunt) {
     }
   });
 
+  // Transifex does not understand placeholders, so this task patches all non-en
+  // locales with missing placeholders
+  grunt.registerTask('locale-patch', function(){
+    var en = grunt.file.readJSON('_locales/en/messages.json');
+    grunt.file.recurse('_locales', function(abspath, rootdir, subdir, filename){
+      if (subdir === 'en' || filename !== 'messages.json'){
+        return;
+      }
+      var messages = grunt.file.readJSON(abspath);
+
+      for (var key in messages){
+        if (en[key]['placeholders'] !== undefined && messages[key]['placeholders'] === undefined){
+          messages[key]['placeholders'] = en[key]['placeholders']
+        }
+      }
+
+      grunt.file.write(abspath, JSON.stringify(messages, null, 4) + '\n');
+    });
+  });
+
+  grunt.registerTask('tx', ['exec:tx-pull', 'locale-patch']);
   grunt.registerTask('dev', ['default', 'connect', 'watch']);
   grunt.registerTask('test', ['jshint', 'jscs', 'connect', 'saucelabs-mocha']);
-  grunt.registerTask('default', ['concat', 'sass', 'copy']);
+  grunt.registerTask('default', ['concat', 'sass', 'locale-patch', 'copy']);
```

Comment:
are we sure we want to remove locale-patch from default? It does add some added protection / detection against forgetting to patch the files. If someone forgets to run `grunt tx` and instead runs `tx pull`, then you can't produce a dist without patching, and it will make it pretty clear quite early on they forgot to patch...